recipes/cpython/all/test_package/conanfile.py

```diff
@@ -94,18 +99,18 @@ def build(self):
         cmake.definitions["Python{}_FIND_STRATEGY".format(py_major)] = "LOCATION"
 
         if self.settings.compiler != "Visual Studio":
-            if tools.Version(self._py_version) < tools.Version("3.8"):
+            if Version(self._py_version) < Version("3.8"):
                 cmake.definitions["Python{}_FIND_ABI".format(py_major)] = self._cmake_abi.cmake_arg
 
-        with tools.environment_append(RunEnvironment(self).vars):
+        with legacy_tools.environment_append(RunEnvironment(self).vars):
```

Comment:
This is a new one for me.

If this is the most pragmatic fix om okay with it,
But why not generate an Environment?

Reply:
two thing here:
- I didn't try to update Conan 2.x features, including Environment, only replaced tools
- legacy_tools is most used to identify legacy tools which should be replaced in the future. It's not mandatory, but it help us when using grep command to list our technical debt.

Reply:
Clear, this will keep things manageable for the migration.